tools/nl/embeddings/data/curated_input/bio/sheets_svs.csv

```diff
@@ -1,45 +1,48 @@
-administrationRoute,administrationRoute,"The method by which a drug is administered",,
-dosageForm,dosageForm,"physical form in which a drug is produced and dispensed",,
-antibodyType,antibodyType,"type of antibody",,
-antigenType,antigenType,"type of antigen",,
-chromosomeSize,chromosomeSize,"number of nucleotides in a chromosome",,"Size of chromosome"
-ensemblID,ensemblID,"Ensembl ID",,
-fullName,fullName,"full name of the gene",,
-geneID,geneID,"gene id",,
-ncbiProteinAccessionNumber,ncbiProteinAccessionNumber,"NCBI protein accession number",,
-alleleOrigin,alleleOrigin,"Variant allele origin",,"Origin of variant allele"
-alleleType,alleleType,"The allele of a genetic variant observed within a population",,"Type of allele"
-ncbiDNASequenceName,ncbiDNASequenceName,"NCBI defined segment of DNA sequence name",,"Name used by NIH NCBI to refer to a segment of DNA sequence"
-imageUrl,imageUrl,"url to an image of what the biological specimen looks like",,"what the entity looks like"
-genomicCoordinates,genomicCoordinates,"genomic coordinates",,
-availableStrength,availableStrength,"dose approved for a drug",,
-referenceSNPClusterID<-GeneticVariantGeneAssociation->geneSymbol,GeneticVariantGeneAssociation,"Association between a genetic variant and a gene",,"Gene associated with a genetic variant;genetic variant associated with a gene"
-diseaseOntologyID<-DiseaseGeneAssociation->geneID,DiseaseGeneAssociation,"Association of a disease and a gene",,
\ No newline at end of file
+administrationRoute,administrationRoute,The method by which a drug is administered,,
+dosageForm,dosageForm,physical form in which a drug is produced and dispensed,,
+antibodyType,antibodyType,type of antibody,,
+antigenType,antigenType,type of antigen,,
+chromosomeSize,chromosomeSize,number of nucleotides in a chromosome,,Size of chromosome
+ensemblID,ensemblID,Ensembl ID,,
+fullName,fullName,full name of the gene,,
+geneID,geneID,gene id,,
+ncbiProteinAccessionNumber,ncbiProteinAccessionNumber,NCBI protein accession number,,
+alleleOrigin,alleleOrigin,Variant allele origin,,Origin of variant allele
+alleleType,alleleType,The allele of a genetic variant observed within a population,,Type of allele
+ncbiDNASequenceName,ncbiDNASequenceName,NCBI defined segment of DNA sequence name,,Name used by NIH NCBI to refer to a segment of DNA sequence
+imageUrl,imageUrl,url to an image of what the biological specimen looks like,,what the entity looks like
+genomicCoordinates,genomicCoordinates,genomic coordinates,,
+availableStrength,availableStrength,dose approved for a drug,,
+<-referenceSNPClusterID{typeOf:GeneticVariantGeneAssociation}->geneSymbol,GeneticVariantGeneAssociation,Gene associated with a genetic variant,,
+<-geneSymbol{typeOf:GeneticVariantGeneAssociation}->referenceSNPClusterID,GeneticVariantGeneAssociation,genetic variant associated with a gene,,
```

Comment:
I would recommend dropping a README in this folder to explain these links?

Reply:
done